_posts/2022-04-22-security-debrief.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: 'Security Debrief: April 17-23, 2022'
+author_avatar: https://avatars.githubusercontent.com/u/2421172?v=4
+author_name: Alex Smith
+author_github_username: tradesmanhelix
+excerpt: A look back at the top security stories from the week.
+---
+
+# This Week in Cybersecurity: April 17-23, 2022
+
+##  Major News Stories
+* As we all know, Heroku had a fun week.
+  * Apparently, it all started when someone at Github noticed that an AWS key stored in a private Github repo was used in an unauthorized manner.
+  * It appears that the issue lies with Heroku, and it likely has something to do with Heroku's use of TravisCI.
+  * The Heroku / Gihub connection is still down as of the time of this post.
+  * Heroku is posting incident status updates here: https://status.heroku.com/incidents/2413
```

Comment:
@tradesmanhelix , unfortunately the markup needs to make this a link explicitly. 

So you will need to do something like `[https://status.heroku.com/incidents/2413](https://status.heroku.com/incidents/2413)` to make this a link I believe.